test/e2e/apps/rc.go

```diff
@@ -64,7 +64,13 @@ var _ = SIGDescribe("ReplicationController", func() {
 		testRCAdoptMatchingOrphans(f)
 	})
 
-	It("should release no longer matching pods", func() {
+	/*
+		Release : v1.13
+		Testname: Replication Controller, release pods
+		Description: Replication Controller Should release no longer matching pods, By creating a given Replicatio Controller. Pod should released, by using PollImmediate by checking the label of the pod .
```

Comment:
s/Should/MUST/

A Replication Controller is created, and its Pods are created.  When the labels on one of the Pods change to no longer match the Replication Controller's label selector, the Replication Controller MUST release the Pod and update the Pod's owner references.

Reply:
updated test description and added tag

>  [Conformance]